test/distrib/bazel/test_single_bazel_version.sh

```diff
@@ -51,12 +51,31 @@ FAILED_TESTS=""
 export OVERRIDE_BAZEL_VERSION="$VERSION"
 # when running under bazel docker image, the workspace is read only.
 export OVERRIDE_BAZEL_WRAPPER_DOWNLOAD_DIR=/tmp
-bazel build -- //... "${EXCLUDED_TARGETS[@]}" || FAILED_TESTS="${FAILED_TESTS}Build "
+
+# pick the Bazel wrapper
+BAZEL="$(pwd)/tools/bazel"
```

Comment:
This now creates an undocumented requirement that this script be run from a particular directory. I think it'd be better if we could avoid that.